code/game/atoms.dm

```diff
@@ -325,7 +319,7 @@
 /atom/proc/blob_act(obj/structure/blob/B)
 	SEND_SIGNAL(src, COMSIG_ATOM_BLOB_ACT, B)
 
-/atom/proc/fire_act(datum/gas_mixture/air, exposed_temperature, exposed_volume, global_overlay = TRUE)
+/atom/proc/fire_act(datum/gas_mixture/air, exposed_temperature = 0, exposed_volume = 0, global_overlay = TRUE)
```

Comment:
this means that you're now defaulting to changing their temperature to `0`; should anything call that without args, it's going to result in weirdness--again, blank args are fine sometimes.